src/server/scripts/Commands/cs_reset.cpp

```diff
@@ -296,6 +314,447 @@ class reset_commandscript : public CommandScript
+    {
+        Player* targetPlayer = GetPlayerFromIdentifierOrSelectedTarget(handler, target);
+
+        if (!targetPlayer)
+        {
+           return false;
+        }
+        else
+        {
+
+            // Delete all items destinations
+            int16_t deletedItemsEquipedCount = resetItemsEquiped(targetPlayer);
+            int16_t deletedItemsInBagsCount = resetItemsInBags(targetPlayer);
+            int16_t deletedItemsInBankCount = resetItemsInBank(targetPlayer);
+            int16_t deletedItemsInKeyringCount = resetItemsInKeyring(targetPlayer);
+            int16_t deletedItemsInCurrenciesListCount = resetItemsInCurrenciesList(targetPlayer);
+            int16_t deletedItemsInVendorBuyBackTabCount = resetItemsInVendorBuyBackTab(targetPlayer);
+            int16_t deletedItemsStandardBagsCount = resetItemDeleteStandardBags(targetPlayer);
+            int16_t deletedItemsBankBagsCount = resetItemDeleteBankBags(targetPlayer);
+
+            handler->PSendSysMessage(LANG_COMMAND_RESET_ITEMS_ALL_BAGS, handler->GetNameLink(targetPlayer).c_str(),
+                deletedItemsEquipedCount,
+                deletedItemsInBagsCount,
+                deletedItemsInBankCount,
+                deletedItemsInKeyringCount,
+                deletedItemsInCurrenciesListCount,
+                deletedItemsInVendorBuyBackTabCount,
+                deletedItemsStandardBagsCount,
+                deletedItemsBankBagsCount);
+        }
+
+        return true;
+    }
+
+private:
+
+    static Player* GetPlayerFromIdentifierOrSelectedTarget(ChatHandler* handler, Optional<PlayerIdentifier> target)
+    {
+        Player* targetPlayer = nullptr;
+
+        // Check if there is an optional target player name
+        // Do not use TargetOrSelf, we must be sure to select ourself
+        if (!target)
+        {
+            // No optional target, so try to get selected target
+            targetPlayer = PlayerIdentifier::FromTarget(handler)->GetConnectedPlayer();
+
+            if (!targetPlayer)
+            {
+                // No character selected
+                handler->SendSysMessage(LANG_NO_CHAR_SELECTED);
+            }
+
+        }
+        else
+        {
+            targetPlayer = target->GetConnectedPlayer();
+
+            if (!targetPlayer || !target->IsConnected())
+            {
+                // No character selected
+                handler->SendSysMessage(LANG_PLAYER_NOT_EXIST_OR_OFFLINE);
+            }
+        }
+
+        return targetPlayer;
+    }
+
+    static int16_t resetItemsEquiped(Player* playerTarget)
+    {
+        if (!playerTarget)
+        {
+            return -1;
+        }
+
+        int16_t count = 0;
+        for (uint8 i = EQUIPMENT_SLOT_START; i < EQUIPMENT_SLOT_END; ++i)
+        {
+            if (Item* pItem = playerTarget->GetItemByPos(INVENTORY_SLOT_BAG_0, i))
+            {
+                if (pItem->GetEntry() == ITEM_EARTH_STONE)
```

Comment:
ITEM_HEARTHSTONE ?

Reply:
> Nevermind

Just to be clear, the typos I explicitly pointed out (that you've corrected) were just the first instances I saw of the typos. There are more cases of the typos that also need to be corrected throughout the changed files.

Reply:
I made a whole replace